lesson10/index.js

```diff
@@ -0,0 +1,77 @@
+const imageArray = [
+    {
+        img: "1.png", 
+        alt:"画像1", 
+        text: "ブックマーク"
+    }, 
+    {
+        img: "2.png", 
+        alt:"画像2", 
+        text: "メッセージ"
+    }
+];
+
+const ul = document.getElementById('js-list');
+
+const createLoading = () => {
+    const li = document.createElement("li");
+    const loadingImg = document.createElement('img');
+
+    li.style.listStyle = "none";
+    li.id = "js-loading";
+    loadingImg.src = 'loading-circle.gif';
+    ul.appendChild(li).appendChild(loadingImg);
+};
+
+const removeLoading = () => {
+    document.getElementById('js-loading').remove();
+};
```

Comment:
IMO and NITS
If the function only returns an expression, the parentheses could be omitted.
I'll leave it up to you to decide if you want to fix this one, since it means I would omit it.
```js
const removeLoading = () => document.getElementById('js-loading').remove();
```

Reply:
Thank you for review!
Nice concise writing!
I also wrote it without the parentheses.
[5e17e98](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/10/commits/5e17e98b0fbe98a0b9f6afc7b17471ab1e74afaa)